Engine/source/console/console.cpp

```diff
@@ -1636,22 +1637,24 @@ static ConsoleValue _internalExecute(SimObject *object, S32 argc, ConsoleValue a
    }
 
    StringTableEntry funcName = StringTable->insert(argv[0].getString());
-
-   const char** argv_str = static_cast<const char**>(malloc((argc - 2) * sizeof(char *)));
-   for (int i = 0; i < argc - 2; i++)
+   if (argc > 2)
    {
-      argv_str[i] = argv[i + 2].getString();
-   }
-   bool result;
-   const char* methodRes = CInterface::CallMethod(object, funcName, argv_str, argc - 2, &result);
+      const char** argv_str = static_cast<const char**>(malloc((argc - 2) * sizeof(char*)));
```

Comment:
Do we not have a typed malloc function for this? Might be cleaner

Reply:
not at time of writing